### DISCUSSION TITLE
Balances panel on Home Dashboard

### DISCUSSION THREAD

**Discussion by pashagolub** [ananthakumaran/paisa#145] [Ideas]
Thanks again for a great product.

The main question every finance app should answer first is how much money (assets) I have on my accounts? Right now I need to go to Assets -> Balances which is two clicks. And the page itself is hard to analyze because of grid structure with blank cells. The information I need is in the Market Value column, the 4th by order.

Net worth & Net Investment are fun to have and to know but they are useless if I want to know from what card to pay for airfare tickets, for example. One more thing is I have no idea what XIRR is and it's documented nowhere :)
![image](https://github.com/ananthakumaran/paisa/assets/9463113/54665e6e-dc27-41f3-9371-9f2b6edebd38)

I propose to have a panel with all checking accounts and their balances top most. Screenshot from [another finance app](https://play.google.com/store/apps/details?id=com.droid4you.application.wallet) as an example
![image](https://github.com/ananthakumaran/paisa/assets/9463113/98a37557-6580-45e2-88b4-13aa3752b68c)

**Comment by ananthakumaran** [comment]
I agree, I personally look at the Checking Balance legend inside the cash flow card. Will get this done. 

> if I want to know from what card to pay for airfare tickets

I don't get this part, do you mean checking account or do you mean you would check the credit card limit before paying the airfare?

> One more thing is I have no idea what XIRR is and it's documented nowhere :)

It's a standard function that's available in Excel, google it. It's used widely in my country, so thought everyone would know :). In essence, it's the rate of return. The one at the dashboard is for your whole assets, you can see more granular value on the balance page.

**Reply by pashagolub** [reply]
>I don't get this part, do you mean checking account or do you mean you would check the credit card limit before paying the airfare?

Sorry. I have several debit cards and sometimes I prefer to pay with the card with the max balance.

**Reply by pashagolub** [reply]
Yeah, I did google for XIRR. :) Even though I have no idea what value is considered normal

**Reply by pashagolub** [reply]
>I agree, I personally look at the Checking Balance legend inside the cash flow card.

That's a good one, yes. But it only gives you the whole amount. For example, I do finances for whole family which means 15 cards/accounts. And sometimes it's good to know I can pay with wife's card for smth :-D

**Reply by ananthakumaran** [reply]
available on master

**Reply by pashagolub** [reply]
Works like a charm in the browser! Thanks!
